nsls2forge_utils/dashboard.py

```diff
@@ -8,31 +8,25 @@
 from .meta_utils import get_attribute
 
 
-def _extract_org(pkg):
-    try_dev_url = False
-    # get org from home url
-    home_str = get_attribute('about home', pkg, 'nsls-ii-forge')
-    if home_str is None:
-        try_dev_url = True
-    elif 'github.com' in home_str:
-        split_str = home_str.split('/')
+def _extract_github_org_from_url(url):
+    if url is not None and 'github.com' in url:
+        split_str = url.rstrip('/').split('/')
         org = split_str[-2]
         if org == 'github.com':
             org = split_str[-1]
     else:
-        try_dev_url = True
+        return ''
```

Comment:
In this case probably better to return `None` and check for that. Or maybe raise an exception and catch it later, but I am in favor of returning `None`s rather than empty strings.

Reply:
The problem with returning `None` is that it will try to place it in the string during formatting. I'd rather it return an empty string so at the worst it displays a 404 badge instead of raising an exception during execution.

Reply:
Sounds good.